### PR TITLE
Create document list page

### PR DESCRIPTION
**Pull request by OnaMosimege**
Add template, view, and responsive  styling to document list page

### CODE REVIEW

**Review by friedelwolff — CHANGES_REQUESTED** [review]
The pagination isn't working correctly for me. I guess some of that is due to the `documents` vs `page_obj`, but even then I'm not convinced that filtering will continue working with pagination.

File: app/app/views.py [hunk 1]
```diff
@@ -161,8 +161,66 @@ def institution_detail(request, institution_id):
...
+
+        if document_languages.count() < 4:
+            languages_data = ", ".join(sorted(language.name for language in document_languages))
+        else:
+            languages_data = _("Multilingual")
+
+        if document_subjects.count() < 4:
+            subjects_data = ", ".join(sorted([subject.name for subject in document_subjects]))
+        else:
+            subjects_data = _("Multiple subjects")
```
[the remark this answers is not in the document]
Comment (OnaMosimege) [reply app/app/views.py:203]: will do

File: app/app/views.py [hunk 2]
```diff
@@ -161,8 +161,66 @@ def institution_detail(request, institution_id):
...
+                "description": document.description,
+                "url": document.url,
+                "category": document.document_type,
+            }
+        )
+
     context = {
         "current_page": "documents",
+        "page_obj": page_obj,
+        "documents": document_data,
```
[the remark this answers is not in the document]
Comment (OnaMosimege) [reply app/app/views.py:220]: I'll look into this

File: app/templates/app/documents.html [hunk 3]
```diff
@@ -3,5 +3,110 @@
...
+        <a href="{% url 'projects' %}" class="btn btn-secondary mt-2 button-spacing">{% trans "Reset" %}</a>
+      </div>
+  </form>
+</div>
 
-{% endblock content%}
+<br />
+
+<div class="project-list">
+  {% for document in documents %}
```
[the remark this answers is not in the document]
Comment (OnaMosimege) [reply app/templates/app/documents.html:47]: I see, will revisit this

**Review by friedelwolff — APPROVED** [review]
The paginator requires much more work, but I guess we need to merge this to get to other pressing matters.

**Pull request merged**